test/tables.jats_archiving

```diff
@@ -4,35 +4,31 @@
     <p>Demonstration of simple table syntax.</p>
   </caption>
   <table>
-    <col align="right" />
-    <col align="left" />
-    <col align="center" />
-    <col align="left" />
```

Comment:
I thought about keeping this, as it *should* work; however, the Lens reader doesn't respect the `align` attribute in `<col>` elements, so the alignment is always done for each cell individually (same as in the HTML writer).